src/ClientCredentialsOptions.cs

```diff
@@ -3,21 +3,21 @@
 
 namespace Laserfiche.Oauth.Api.Client
 {
-    public class ClientCredentialsOptions : IClientCredentialsOptions
+    public class ClientCredentialsOptions
```

Comment:
Is this class structure suppose to match the js version?
https://github.com/Laserfiche/lf-oauth-api-client-js/blob/1.x/lib/ClientCredentialsOptions.ts

Reply:
Fix.